operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -72,6 +71,27 @@ spec:
 # Choose "anonymous" to allow full access to Kiali without requiring any credentials.
 # Choose "openshift" to use the OpenShift OAuth login which controls access based on
 # the individual's  RBAC roles in OpenShift. Not valid for non-OpenShift environments.
+# Choose "ldap" to enable LDAP based authentication. There are additional configurations for
+# under ldap in the yaml file. The configurations for LDAP are as follows
+#    ---
+#    strategy: ldap
+#    ldap:
+#       # AD IP host and port
```

Comment:
BTW: I assume this stuff will work with other LDAP vendors and not just ActiveDirectory, correct?

Reply:
Yes it will work with other LDAP vendors